pythonforandroid/bootstrap.py

```diff
@@ -263,11 +264,10 @@ def strip_libraries(self, arch):
             info('Python was loaded from CrystaX, skipping strip')
             return
         env = arch.get_env()
-        strip = which('arm-linux-androideabi-strip', env['PATH'])
-        if strip is None:
-            warning('Can\'t find strip in PATH...')
-            return
-        strip = sh.Command(strip)
+        tokens = shlex.split(env['STRIP'])
+        strip = sh.Command(tokens[0])
+        if len(tokens) > 1:
```

Comment:
have you tested this without the `if` and a single argument? I tested bake with an empty list and it worked fine, I'm pretty sure this conditional is not required

Reply:
Oh, you're quite right, I misremembered what list slice syntax would do outside the bounds of the list. It's fine to revert this.

Reply:
@JonasT Yes, I tested the `len(tokens) == 1` case, and it basically works.  But, I found that `.bake()` raises a complaint when it's argument is `[]`  (see the following REPL test).
```
>>> import sh
>>> sh.Command('strip').bake([])
__main__:1: UserWarning: Empty list passed as an argument to '/opt/local/bin/strip'. If you're using glob.glob(), please use sh.glob() instead.
```
If we want to eliminate this warning, I think we still need the `if` check.  Thoughts? @JonasT @inclement

Reply:
@j-devel oh weird, have never seen that (maybe I got a different version?). In that case definitely keep the check in